Insights/Account/UserAccessPermissions.yaml

```diff
@@ -0,0 +1,46 @@
+SchemaVersion: '1.0'
+DataTypes:
+  - DataType: UserAccessAnalytics
+Provider: Sentinel
+Type: KQL
+EntitiesFilter: {}
+BaseQuery: |
+  let GetUserSubscriptionRbac = (v_Account_AadUserId:string) { 
+    let blastRadTime = now();
+    let userAccess = UserAccessAnalytics|where SourceEntityId == v_Account_AadUserId; 
```

Comment:
In general, these are difficult to read, would be good to have a line return for each new statement.

Reply:
agree. added newlines